src/service/comment.js

```diff
@@ -107,14 +107,16 @@ const createChildCommentV3 = async (
       message: text,
       sendPostNotif,
       activityId,
-      anonimity: isAnonymous
+      anonimity: isAnonymous,
+      is_you: true
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `is_you` property is added to the `data` object. This property seems to indicate whether the current user is the one who created the comment. However, it's always set to `true`, which might not be correct if this function is used to create comments for other users as well. Please ensure that this property is correctly set based on the user context.

```diff
-      anonimity: isAnonymous,
-      is_you: true
+      anonimity: isAnonymous,
+      is_you: currentUser === commentAuthor
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->